src/libs/ajax.js

```diff
@@ -76,12 +79,24 @@ const withRetryOnError = _.curry(wrappedFetch => async (...args) => {
   return wrappedFetch(...args)
 })
 
+
+// Captures given course in error message, compares to root addresses in dav.json plus https://storage.googleapis.com/,
+// and returns the root that matches the error.
+const captureRetryFailure = (...args) => {
```

Comment:
```suggestion
const captureRequestFailure = (...args) => {
```

The current name is a bit misleading. It doesn't have anything to do with retries.